### ISSUE TITLE
Crash if setting has no value

### ISSUE THREAD

**Issue by mikeaix** [ByteDev/ByteDev.DotNet#5]
For whatever reason my real life solution file contains a setting with a key but no value:

```
	GlobalSection(SpecExplorer.ActivityCompletionStatus) = preSolution
		SpecExplorer.ActivityCompletionStatus = 
	EndGlobalSection
```

This crashes ByteDev.DotNet.

**Comment by mikeaix** [comment]
I've added a pull request for this issue.

**Comment by ByteDev** [comment]
Fix is now in version 7.2.1 of the package (master branch).

Thanks!
